src/Morphir/IR/SDK/LocalDate.elm

```diff
@@ -35,15 +35,23 @@ moduleName =
     Path.fromString "LocalDate"
 
 
+config =
+    { baseType = stringType ()
+    , toBaseType = toFQName moduleName "toIsoString"
+    , fromBaseType = toFQName moduleName "fromISO"
+    }
+
+
 moduleSpec : Module.Specification ()
 moduleSpec =
     { types =
         Dict.fromList
-            [ ( Name.fromString "LocalDate", OpaqueTypeSpecification [] |> Documented "Type that represents a date concept." )
+            [ ( Name.fromString "LocalDate", DerivedTypeSpecification [] config |> Documented "Type that represents a date concept." )
             ]
     , values =
         Dict.fromList
-            [ vSpec "fromISO" [ ( "iso", stringType () ) ] (maybeType () (localDateType ()))
+            [ vSpec "toIsoString" [ ( "date", localDateType () ) ] (stringType ())
```

Comment:
For abbreviations you should use all upper-case so this should be `toISOString`. See an explanation here: https://github.com/finos/morphir-elm/blob/17eb6250693ea3f6a1f6cf4fa7eb17399eec57a4/src/Morphir/IR/Name.elm#L34-L48

Reply:
Noted. 
will correct this.